rollbar-java/src/main/java/com/rollbar/notifier/sender/json/JsonSerializerImpl.java

```diff
@@ -0,0 +1,191 @@
+package com.rollbar.notifier.sender.json;
+
+import static java.util.regex.Pattern.compile;
+
+import com.rollbar.api.json.JsonSerializable;
+import com.rollbar.api.payload.Payload;
+import com.rollbar.notifier.sender.result.Result;
+import com.rollbar.notifier.sender.result.ResultCode;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Implementation of the {@link JsonSerializer json serializer}.
+ */
+public class JsonSerializerImpl implements JsonSerializer {
```

Comment:
This looks like the code from the JSON serializer from the current rollbar-java, so I am assuming it is from there. I have no idea why that library had a handrolled JSON serializer, but it seems crazy to me. Can we use Jackson or GSON or something standard?

Reply:
Yes, the code from the JsonSerializer and SyncSender it's coming from the previous version. 
I guess that we were using a handrolled JSON serializer to not introduce a third party dependency in our code that would end up being a transitive dependency for our users, but this is something that can be changed.